drafts/2016-01-06.md

```diff
@@ -0,0 +1,91 @@
+* [@jbenet](//github.com/jbenet) released a new tool/library called [dnslink](https://github.com/jbenet/go-dnslink) that makes it easy to resolve dns links (special TXT records in a domain name that can point to paths, like an IPFS path)
+
+### Updates
+* [**(infrastructure)**](https://github.com/ipfs/infrastructure) On the infrastructure side of things, [@lgierth](//github.com/lgierth) has [bootstrapped](https://github.com/ipfs/infrastructure/pull/135) two new storage, each with 17 TB of disk space!  
+* [**(api)**](https://github.com/ipfs/api) [@RichardLitt](//github.com/RichardLitt) has [reached a draft 1]((https://github.com/ipfs/api/pull/13) of the much needed API documentation.
+* [@harlantwood](//github.com/harlantwood) wrote a bit of [nodejs code](https://github.com/ipfs/project-repos/pull/11) that spins up a fresh IPFS node, sets it to a known ID, and publishes to IPNS using that node 
+* [**(specs)**](https://github.com/ipfs/specs) The new IPFS Linked Data (IPLD) spec is actively being iterated on in the specs repository.  Join the discussion [here!](https://github.com/ipfs/specs/pull/37)
+
+### Active stuff
+* [@robcat](//github.com/robcat) and [@fazo](//github.com/fazo)96 have done great work integrating IPFS with pacman (the package manager for Arch Linux).  They can now install arch packages straight from IPFS!  For more details, see [this active discussion](https://github.com/ipfs/notes/issues/84).
+* [@Dignifiedquire](//github.com/Dignifiedquire) has been working on an attractive new distribution page for IPFS, which will be the new landing page to download all things IPFS.  You can see the [latest screenshots here](https://github.com/ipfs/distributions/issues/11).
+
+## Contributors
+
+Across the entire IPFS github organization, the following people have comitted code since December 7th. (We're autogenerating this list using [this tool](https://github.com/ipfs/weekly/pull/5), so please let us know if your name isn't here.) In the future, we will also include people who comment, as they are also super important; bear with us while we develop that technology.
+
+
+* Adam Gashlin
+* Alex Mingoia
+* Christian Couder
+* David Dias
+* Dominic Della Valle
+* Enrico Fasoli
+* Friedel Ziegelmayer
+* Harlan T Wood
+* Ian Preston
+* Ivan Vilata-i-Balaguer
+* Jakub (Kubuxu) Sztandera
+* Jeromy Johnson
+* Jonathan Dahan
+* Juan Benet
+* Kyle Drake
+* Lars Gierth
+* Lars-Magnus Skog
+* RainerWasserfuhr
+* ReadmeCritic
+* Richard Littauer
+* Simon Kirkby
+* Teo Sartori
+* Travis Person
+* digital dreamer
+* dignifiedquire
+* greenkeeperio-bot
+* rht
+
+Thanks, and see you next week!
+
+- Richard Littauer and Andrew Chin
+
+[Feedback about the Weekly](https://github.com/ipfs/weekly/issues/7)
```

Comment:
maybe "Send us feedback about the Weekly" (so that it's a clear action, instead of "here's where i can _read_ feedback)